hr_utilization_analysis/report/hr_utilization_analysis.py

```diff
@@ -52,18 +52,88 @@ def _compute_entry_ids(self):
         for analysis in self:
             employee_ids = HrEmployee.search(analysis._get_employees_domain())
 
-            entry_ids = [(5, False, False)]
-            for employee_id in employee_ids:
-                date = analysis.date_from
-                one_day = timedelta(days=1)
-                while date <= analysis.date_to:
-                    entry_ids.append(
-                        (0, False, {"employee_id": employee_id.id, "date": date})
-                    )
-                    date += one_day
+            dates = []
+            date = analysis.date_from
+            one_day = timedelta(days=1)
+            while date <= analysis.date_to:
+                dates.append(date)
+                date += one_day
+
+            entry_ids = [(5, False, False)] + [
+                (0, False, d) for d in self._get_entry_id_dicts(employee_ids, dates)
+            ]
 
             analysis.entry_ids = entry_ids
 
+    def _get_entry_id_dicts(self, employees, dates):
```

Comment:
It feels more like `_get_entries` in general

Reply:
Noting about ID really anymore

Reply:
What about `_get_entry_values`